ark/ark.py

```diff
@@ -79,6 +79,7 @@ def transactions(self):
             raise NotImplementedError()
 
     def transport(self):
+        self.port = 4002
```

Comment:
why set it here and not in the `__init__`?

Reply:
It is already set in init. This is to reset the variable and switch to 4002 if you hit a corev2 node with V1 API. Otherwise API call fails. In future this would come from a configuration vs hard-coded switch

Reply:
Added updated install instruction per suggestion